src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostExtensions.cs

```diff
@@ -109,5 +111,46 @@ public static IHostBuilder UseExternalStartup(this IHostBuilder builder, IWebJob
 
             return builder;
         }
+
+        public static IHostBuilder ConfigureWebJobsFastLogging(this IHostBuilder builder, IEventCollectorFactory fastLogger)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            if (fastLogger == null)
+            {
+                throw new ArgumentNullException(nameof(fastLogger));
+            }
+
+            builder.ConfigureServices(services =>
+            {
+                services.AddWebJobsFastLogging(fastLogger);
+            });
+
+            return builder;
+        }
+
+        /// <summary>
+        /// Adds the ability to bind to an <see cref="ExecutionContext"/> from a WebJobs function.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <param name="configure">An optional <see cref="Action{ExecutionContextBindingOptions}"/> to configure the provided <see cref="ExecutionContextOptions"/>.</param>
+        /// <returns></returns>
+        public static IHostBuilder AddExecutionContextBinding(this IHostBuilder builder, Action<ExecutionContextOptions> configure = null)
```

Comment:
Debated not having the `AddExecutionContextBinding` and just adding the execution context binding with the core ones. Thoughts?

Reply:
👍